core/plugins/org.polarsys.capella.core.sirius.analysis/src/org/polarsys/capella/core/sirius/analysis/refresh/extension/InteractionRefreshExtension.java

```diff
@@ -65,7 +69,7 @@ public void beforeRefresh(DDiagram diagram) {
   @Override
   public void postRefresh(DDiagram diagram) {
     clearCaches();
-    super.postRefresh(diagram);
+    isCacheEnabled = false;
```

Comment:
We could condition the putxxxCache method to have an effect only in this instead of check theisCacheEnabled value everywhere.

Teh need to have the isCacheEnabledMethod would be less usefull.

Reply:
If we remove the super.beforeRefresh / afterRefresh method could you confirm that there is another extension which enable the ModelCache during the SequenceDDiagram refresh ? 

If there is another extension doing this, we can remove the superbeforeR/afterR methods but in this case we should also remove the inheritance to the AbstractCacheAwareRefreshExtension.

It seems that the TitleBlockRefreshExtensionProvider is enabled for every diagram.

Reply:
done